spec-0014/index.md

```diff
@@ -0,0 +1,369 @@
+
+caption = "A screenshot of an interactive example in the NumPy documentation. The blue button labeled 'Try it in your browser!' allows users to run the code in a JupyterLite kernel directly within the documentation page without leaving the site."
+
+title = "An interactive example for the numpy.hsplit() function"
+
+{{< /figure >}}
+
+---
+
+{{< figure >}}
+
+src = "./NumPy-interactive-example-opened.png"
+alt = "This image shows the interactive JupyterLite notebook that has been opened and executed after clicking the 'Try it in your browser!' button from the NumPy hsplit documentation. The notebook is running directly within the same documentation page in an iframe, allowing users to experiment with the function without leaving the documentation. The JupyterLite UI is visible with standard notebook controls (File, Edit, View, Run, Kernel, Settings, Help) at the top. Below that is a warning message in an orange box stating that 'NumPy's interactive examples are experimental and may not always work as expected, with high load times, especially on low-resource platforms, and the version of NumPy might not be in sync with the one you are browsing the documentation for. The notebook shows three executed code cells, demonstrating how numpy.hsplit works with different parameters. The first cell imports NumPy and creates a 4x4 array, the second cell applies hsplit(x, 2) to split the array into two equal parts, and the third cell shows hsplit(x, np.array([3, 6])) to split the array at specific column indices."
+
+caption = "The interactive JupyterLite notebook that has been opened and executed after clicking the 'Try it in your browser!' button from the NumPy hsplit documentation. The notebook is running directly within the same documentation page in an iframe, allowing users to experiment with the function without leaving the documentation."
+title = "Running the interactive example within the documentation page"
+
+{{< /figure >}}
+
+### Core Project Endorsement
+
+Core projects endorsing this SPEC would mean that they:
+
+1. enable interactivity within applicable sections of their documentation websites, such as
+   - the API reference sections (e.g., for "Examples" sections for public API methods and classes)
+   - narrative long-form content (tutorials, how-to guides, etc., that are usually written in the form of notebooks)
+2. further encourage other Scientific Python projects to adopt similar interfaces to make their documentation websites interactive, and
+3. take interactive documentation deployments into account when making decisions about the future of their documentation, and
+4. consider the suggestions made in this SPEC when doing so, wholly or in part.
+
+### Ecosystem Adoption
```

Comment:
What do you think of linking to some documentation pages (Numpy, scikit-learn) that already have this setup so that people can "witness the miracle" with just a few clicks? Basically, showing them how cool interactive docs are by linking them to a few good examples. A bit like itneractive docs let users of that library witness the miracle of the library :D

We could then even go a step further and in the section(s) on "how to do this for your project" link to how things are implemented for the examples linked to here.